Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Models/PrimerCardValidationModels.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  PrimerCardValidationModels.swift
+//  PrimerSDK
+//
+//  Created by Jack Newcombe on 30/10/2023.
+//
+
+import Foundation
+
+@objc
+public class PrimerCardState: NSObject {
+    public let cardNumber: String
```

Comment:
If this object represents the current string from the card number text field (not complete, but while the user is typing), that has to be validated, maybe consider a different name. Here is what GPT suggests :) https://chat.openai.com/share/3ff353ef-09a3-46b0-b4ea-1a01aa097b82

Reply:
Do you mean for the class name or the property name? I was previously using `cardFragment` for the property name but changed it to `cardNumber` as it clashed with the naming for existing local valdiation

@semirp what are you using in this case?

Reply:
Class name... But it could be both for more context, class and property names could be different... Not a must, just a nice-to-have

Reply:
`CardNumberEntryState` looks good - I think this with a `Primer` suffix 👍